src/links/v1/providers.json

```
@@ -123,8 +123,8 @@
       "id": "matterverse",
       "attributes": {
         "name": "Matterverse",
-        "description": "Database of 31 million hypothetical materials generated through advanced machine learning algorithms",
-        "base_url": "https://providers.optimade.org/index-metadbs/matterverse",
+        "description": "Database of M3GNet predicted properties for over 31 million hypothetical materials and all materials in the Materials Project.",
+        "base_url": "https://optimade.matterverse.ai",
```

@JiQi535 Thanks for updating the entry! The policy for ` src/links/v1/providers.json ` is that it should link to an "index meta-database" so there is always one level of indirection to real data (which helps when providers later realize they want to expand into providing more than a single database). Hence, either just keep `"https://providers.optimade.org/index-metadbs/matterverse` in `providers.json` (which thus points to that index metadb, which then points at your URL), *or* also host an index meta-database somewhere under matterverse, e.g., `https://optimade-index.matterverse.ai`.

(Pinging @ml-evs to correct me if I'm saying anything wrong)

```suggestion
        "base_url": "https://providers.optimade.org/index-metadbs/matterverse",
```

---

Thanks for the commit suggestion and nice explanation. Now I understand the purpose of this base_url in providers.json.